Python/bytecodes.c

```diff
@@ -4040,17 +4040,14 @@ dummy_func(
             DEOPT_IF(callable_o != (PyObject *)&PyUnicode_Type);
         }
 
-        op(_CALL_STR_1, (callable, null, arg -- res)) {
+        op(_CALL_STR_1, (callable, null, arg -- res, a)) {
             PyObject *arg_o = PyStackRef_AsPyObjectBorrow(arg);
 
             assert(oparg == 1);
             STAT_INC(CALL, hit);
             PyObject *res_o = PyObject_Str(arg_o);
-            DEAD(null);
-            DEAD(callable);
-            (void)callable; // Silence compiler warnings about unused variables
-            (void)null;
-            PyStackRef_CLOSE(arg);
+            a = arg;
+            INPUTS_DEAD();
             ERROR_IF(res_o == NULL);
             res = PyStackRef_FromPyObjectSteal(res_o);
```

Comment:
It should be like this:
```suggestion
            if (res_o == NULL) {
                ERROR_NO_POP();
            }
            a = arg;
            INPUTS_DEAD();            
            res = PyStackRef_FromPyObjectSteal(res_o);
```
The reason is that ERROR_IF path might leak the reference count of `arg` as we mark it as dead. However, if we leave it on the stack, the error handler will later decref it for us.